doc/workflow/coding_standard_qml.md

```diff
@@ -0,0 +1,175 @@
+# Coding Standard QML
+
+###1. QML document structure
+
+#####Rule 1.1. Basic document and elements structure should be order like this:
```

Comment:
So what about my commet above (about rule codes)? Or you prefer to leave existing names for the rules?